bouncer/shared/lp_api_test.ts

```diff
@@ -167,11 +175,17 @@ async function testTransferAsset() {
   // Expect result to be a block hash
   assert.match(result, /0x[0-9a-fA-F]{64}/, `Unexpected transfer asset result`);
 
-  const newBalance = JSON.stringify(
-    await chainflip.query.liquidityProvider.freeBalances(destinationLpAccount.address, testAsset),
-  );
+  const newBalancesSource = await getLpBalance(sourceLpAccount.address);
+  const newBalanceDestination = await getLpBalance(destinationLpAccount.address);
 
-  assert(newBalance > oldBalance, `Failed to observe balance increase after transfer`);
+  assert(
+    newBalanceDestination > oldBalanceDestination,
+    `Failed to observe balance increase after transfer for destination account!`,
+  );
+  assert(
+    newBalancesSource < oldBalanceSource,
+    `Failed to observe balance decrease after transfer for source account!`,
+  );
```

Comment:
Should the balances be equal before and after?

Ie. can we improve this check even further and test for `oldBalanceSource + oldBalanceDestination === newBalancesSource + newBalanceDestination` ?